pkg/provider-local/admission/validator/cloudprofile.go

```diff
@@ -48,9 +48,15 @@ func (cp *cloudProfileValidator) Validate(_ context.Context, newObj, _ client.Ob
 	if err != nil {
 		return fmt.Errorf("could not decode providerConfig of CloudProfile %q: %w", cloudProfile.Name, err)
 	}
+
 	capabilityDefinitions, err := helper.ConvertV1beta1CapabilityDefinitions(cloudProfile.Spec.MachineCapabilities)
 	if err != nil {
 		return field.InternalError(field.NewPath("spec").Child("machineCapabilities"), err)
 	}
+
+	if err := validation.ValidateSupportedCapabilities(capabilityDefinitions, field.NewPath("spec").Child("machineCapabilities")); err != nil {
```

Comment:
Can you elaborate on why we need this restriction in the provider local, i.e. why is it only allowed to use the architecture capability?